src/main/resources/init_db.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS books(
+    id BIGINT AUTO_INCREMENT PRIMARY KEY,
+    title VARCHAR(255) NOT NULL,
+    price DECIMAL(10, 2) NULL
+)
```

Comment:
![image](https://github.com/mate-academy/jv-jdbc-intro/assets/49767049/913d64ea-f666-4ea4-85f3-0aaf7d208541)
don't forget to always leave that one empty line at EOF